src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Hosting/ConfigurableLanguageServerFeatureOptions.cs

```diff
@@ -60,6 +62,7 @@ public ConfigurableLanguageServerFeatureOptions(string[] args)
             TryProcessBoolOption(nameof(UseRazorCohostServer), ref _useRazorCohostServer, option, args, i);
             TryProcessBoolOption(nameof(DisableRazorLanguageServer), ref _disableRazorLanguageServer, option, args, i);
             TryProcessBoolOption(nameof(ForceRuntimeCodeGeneration), ref _forceRuntimeCodeGeneration, option, args, i);
+            TryProcessBoolOption(nameof(UseRoslynTokenizer), ref _useRoslynTokenizer, option, args, i);
```

Comment:
What happens if the options result in `UseRoslynTokenizer && !ForceRuntimeCodeGeneration` being true? Is that a configuration that we support?

Reply:
I wasn't aware of any relationship between the two features?

It would definitely be possibly to hit though, as both of those flags are controlled by VS Code user settings.

Reply:
News to me, but makes sense. I guess we need @333fred to confirm, and I'll make the appropriate updates to the logic in the IDE. Or is it automatically handled by the compiler?

Reply:
> I thought that we tried to simplify the overall scenarios a bit by saying that users could only use the new tokenizer if FUSE was enabled.

Sorry, that's news to me too. I don't remember us saying that.

Reply:
Yeah, the two features should be unrelated, and you can have one on without the other.